### PR TITLE
{cae}[foss/2023a] OpenFOAM-Extend v4.1-20210705

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3105.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/95377c90e5e6d0f7793e8e78d5b4b81a for a full test report.

File: easybuild/easyconfigs/o/OpenFOAM-Extend/OpenFOAM-Extend-4.1-20210705_fix-private-PackedList.patch [hunk 1]
```diff
@@ -0,0 +1,14 @@
+fix for:
```
Comment (laraPPr) [inline easybuild/easyconfigs/o/OpenFOAM-Extend/OpenFOAM-Extend-4.1-20210705_fix-private-PackedList.patch:1]: Is there something missing in this line?

**Review by laraPPr — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/o/OpenFOAM-Extend/OpenFOAM-Extend-4.1-20210705_fix-private-PackedList.patch (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/o/OpenFOAM-Extend/OpenFOAM-Extend-4.1-20210705_fix-private-PackedList.patch:1]: No, see next line

**Review by laraPPr — APPROVED** [review]
lgtm

**Comment by laraPPr** [comment]
Going in, thanks @boegel!

**Pull request merged**